task_5/config/index.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  port: 3000,
```

Comment:
config should be hardcoded, plus there are packages to manage settings on different levels - file, env vars etc

Reply:
done